design/fake-client.rst

```diff
@@ -63,16 +67,22 @@ Design
 In the following description, the second design option is used.
 
 There is a fake session object that the user of the zhmcclient package
-will use instead of zhmcclient.Session. The fake session object contains a
+will use instead of `zhmcclient.Session`. The fake session object contains a
 faked HMC that can represent all resources relevant for the zhmcclient.
-There fake session object implements the same API as zhmcclient.Session, so
-that the rest of the zhmcclient classes does not need to be changed.
-The get(), post() and delete() methods of the fake session class will
+
+This fake session object implements the same API as `zhmcclient.Session`, so
+that the rest of the zhmcclient classes do not need to be changed.
+The `get()`, `post()` and `delete()` methods of the fake session class will
```

Comment:
The (slight?) disadvantage of that approach is, that the testing surface is larger. We should give it a try though.